modules/tinymce/changelog.txt

```diff
@@ -1,5 +1,5 @@
 Version 5.7.0 (TBD)
-
+    Checking for empty url before calculating image dimensions #TINY-6611
```

Comment:
`Checking` isn't a valid changelog entry. It must be one of the following:
- Added
- Changed
- Deprecated
- Fixed
- Removed

Reply:
Done